### ISSUE TITLE
Strength modifier not aplied to melee weapon damage

### ISSUE THREAD

**Issue by johcker** [cyface/foundryvtt-dcc#64]
I was trying a character with 18 strength and noticed that the strength modifier is not being applied to the damage in melee combat. I think it's another bug.

**Comment by mooped** [comment]
The vast majority of the boxes for modifiers to rolls are 'direct reading'. It rolls the dice plus whatever is in the box.

While it would be convenient to automatically calculate everything it creates a lot of behind the scenes complexity and can be confusing for the user to understand. It also creates problems when people want to use the sheet for custom classes that bend the rules a bit, and for classes that have specific bonuses.

Right now I believe this behaviour is consistent with the rest of the sheet, with the exception of the luck modifier being factored automatically into crit and fumble rolls.

**Comment by cyface** [comment]
I agree. You can put your strength mod into the damage box on the weapon.

**Comment by johcker** [comment]
I have to disagree about this question. I understand that behind the scenes a lot of controls and complexity exists, however a basic rule (Combat Attack and Combat Damage) wouldn't be a problem.

I really appreciate your work with this system. Few months ago I created a [sheet to a game in Roll20](https://github.com/Roll20/roll20-character-sheets/tree/master/Old%20Dragon%20-%20Aprimorado) and after 8000 lines of code, some people  kept to complain about aesthetic things (). I kept my sanity because I made a design rule: the main and important rules of the system have to be maintained and improved, but do not to listen to the homebrew rules that bring unnecessary complexity. 

Applying the str modifier to damage rolls is an easy thing to do and can avoid some problems for the GM in a game that he will have to deal with burning attribute points. The modifier will float during the game, so the player will need to modify (and the GM to track it) all the time.
